tools/testsys/src/run.rs

```diff
@@ -170,14 +229,20 @@ pub(crate) enum TestType {
     /// variance this will run sonobuoy in "quick" mode. For ECS variants, this will run a simple
     /// ECS task.
     Quick,
+    /// Migration testing ensures that all bottlerocket migrations work as expected. This requires
+    /// `migration-starting-version` and `migration-target-version`. To target a specific starting
+    /// ami use `starting-image-id`. A migration test starts by creating necessary instances, then
+    /// running a quick conformance test. Then the nodes are upgraded to the
+    /// `migration-target-version` and a second quick test is run. Lastly, the nodes are downgraded
+    /// back to `migration-starting-version`.
```

Comment:
nit: this comment can just be a high-level summary of what migration testing is, rather than the required arguments or the specific test sequence.